Gemfile.lock

```diff
@@ -45,6 +45,7 @@ GEM
     aws-sigv4 (1.2.1)
       aws-eventstream (~> 1, >= 1.0.2)
     babosa (1.0.3)
+    bigdecimal (2.0.0)
```

Comment:
Because of this version, I'm getting this error

```
snmacos  release/1.14 [2$] 💎 v2.5.5
➜ export PUBLIC_CONFIG_FILE="./config/Version.Public.xcconfig" && export PROJECT_ROOT_FOLDER="./" && bundle exec fastlane run ios_bump_version_beta
[✔] 🚀
[14:28:29]: Error loading plugin 'fastlane-plugin-wpmreleasetoolkit': undefined method `new' for BigDecimal:Class
```

Despite that, `bundle exec fastlane run ios_bump_version_beta` does what it's supposed to. I think the reason is that it doesn't call `BigDecimal.new` directly.

But, if we try to run a lane, the fact that the plugin cannot be loaded means the lane will fail. This can be verified by running `bundle exec fastlane get_pullrequests_list`.

I left a comment in the sister PR on how to fix this.